src/config.rs

```diff
@@ -121,6 +121,12 @@ pub struct Config {
     /// The number of ms for timeouts when publishing messages to kafka.
     pub kafka_send_timeout_ms: u64,
 
+    /// The url of the postgres database to use for the inflight activation store.
+    pub pg_url: String,
+
+    /// The name of the postgres database to use for the inflight activation store.
+    pub pg_database_name: String,
```

Comment:
You should be able to combine these, and include the database name into the url.

Reply:
I separated it because technically I interact with 2 databases: default, to create the DB, and then the actual DB to create the table.

Reply:
Right, because you can't connect to a database that doesn't exist, and you need to create the db initially.